deploy/cloud/modules/provision-apps/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "namespace" {
+  type = string
+  default = "kube-system"
```

Comment:
why not `default = "kube-system"` for an nginx deployment?
deploying into `kube-system` isn't usual.

Reply:
fixed